sample-hip/specs/hip-facade.yaml

```diff
@@ -186,6 +186,8 @@ components:
     FacadeResponse:
       type: object
       properties:
+        clientRequestId:
+          type: String
```

Comment:
if we are making any change in spec, then we should generate corresponding code and raise pr

Reply:
Generated sample files